src/lib/setup.js

```diff
@@ -61,3 +61,25 @@ export async function initGit(target) {
 		warn('Could not locate `git` binary in `$PATH`. Skipping!');
 	}
 }
+
+// Formulate Questions if `create` args are missing
+export function isMissing(argv) {
+	let out = [];
+
+	const ask = (name, message, val) => {
+		let type = val === void 0 ? 'input' : 'confirm';
+		out.push({ name, message, type, default:val });
+	};
+
+	// Required data
+	!argv.template && ask('template', 'Remote template to clone (user/repo#tag)');
```

Comment:
Better to show the default value here. Like what we did for `preact init`?

Reply:
basically its like having a fallback value for all the prompts

Reply:
There is no default value -- it's required.